data-manipulation/hashing/murmur/hash-data-using-murmur3.yml

```diff
@@ -28,3 +28,21 @@ rule:
         - and:
           - number: 0x87c37b91114253d5 = 64-bit c1 for 128-bit hash
           - number: 0x4cf5ad432745937f = 64-bit c2 for 128-bit hash
+        - and:
+          - basic block:
+            - and:
+              - mnemonic: rol
+              - number: 15 = r1
+              description: k ROL r1
+          - basic block:
+            - and:
+              - mnemonic: rol
+              - number: 13 = r2
+              description: hash ROL r2
+        - basic block:

```

Comment:
is this indented correctly?

Reply:
yes, this block could be in a different function than the other two. The reason why I put the other two together is that `ROL 13` and `ROL 15` are not that unique and will probably cause false positives. But together this should be unique enough.

Should I add a comment or clarify this somehow?

Reply:
as is, this block will be a stand-alone detection though (its grouped under the top-level `or`)

I'm not sure about using `3 or more` for the count. I'd probably leave it for now.

Reply:
@mr-tz 
> as is, this block will be a stand-alone detection though (its grouped under the top-level or)

yes, do you think it shouldn't? I think finding `hash >> 16; hash >> 13; hash >> 16` is enough to say this is most likely Murmur Hashing.

Reply:
Ok, in that case we're good. Thanks for the clarifications.

Reply:
I'll add a comment so that it is clear.